### DISCUSSION TITLE
SimNIBS with extracephalic return placement

### DISCUSSION THREAD

**Discussion by drschr** [simnibs/simnibs#252] [8. Simulation workflows]
Dear SimNIBS team,

I am a tDCS researcher with psychological background from Southwest Germany.
I am reaching out to you because we frequently use extracephalic montages in tDCS (i.e., return electrode placed at upper arm to avoid polarization of another brain region). We also like to employ electric field modeling and found SimNIBS to become ever more powerful and helpful for our research. However, we currently do not have a good solution for modeling the extracephalic montage with SimNIBS.

I know that this is a very special case, but I was wondering if you might already have a solution in place for realistic electric field computations of this montage in SimNIBS? At the moment, we often approximate this by placent the return electrode at the most distant position available in the head model. However, we do know that electrode distance may yield smaller motor evoked potentials (Moliadze et al., 2010), suggesting that a larger distance would result in smaller modulation (after-)effects.

Do you have any suggestions how we can improve the modeling for our electrode montage? Any hints would be highly appreciated.

Thanks for your consideration and for the continuous effort with SimNIBS,

Philipp (Schroeder)

**Comment by TorgeW** [comment]
This is unfortunately not yet possible.
